symplyphysics/laws/electricity/electromotive_force_induced_in_uniformly_rotating_coil.py

```diff
@@ -22,74 +22,54 @@
 #. The angular speed of the coil's rotation constant.
```

Comment:
What's the difference between this law and sinusoidally rotating coil? Descriptions are almost identical and formulas are the same.

Reply:
They're the same. I'll remove one.